release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1968,6 +2011,16 @@ Workaround: Manually or by using a daemon set, modify the file `/etc/cni/tuning/
 
 * The Cluster Network Operator (CNO) controller is monitoring more resources than it needs to. As a result, its reconciler is being triggered too frequently, causing a much higher rate of API requests than necessary. There is approximately 1 config map access request made every second. This increases the load on both the CNO and the `kube-apiserver`. (link:https://issues.redhat.com/browse/OCPBUGS-11565[*OCPBUGS-11565*])
 
+[id="ocp-4-13-ran-known-issues"]
+* If you specify an invalid subscription channel in the subscription policy that you use to perform a cluster upgrade, the {cgu-operator-first} indicates that the upgrade is successful immediately after {cgu-operator} enforces the policy because the `Subscription` resource remains in the `AtLatestKnown` state.
+(link:https://issues.redhat.com/browse/OCPBUGS-9239[*OCPBUGS-9239*])
+
+* `kdump` fails to generate the `vmcore` crash dump file on HPE Edgeline e920t servers that are running the real-time kernel with Intel E810 NIC and ice driver.
+(link:https://issues.redhat.com/browse/RHELPLAN-138236[*RHELPLAN-138236*])
+
+* In {ztp}, when you provision a managed cluster that contains more than a single node using a `SiteConfig` CR, disk partition fails when one or more nodes has a `diskPartition` resource configured in the `SiteConfig` CR.
```

Comment:
@pixelsoccupied can you Can you LGTM on this 4.13 known issue text please.

Reply:
LGMT thanks!